assembl/static2/js/app/utils/translations.js

```diff
@@ -1787,6 +1792,11 @@ const Translations = {
       slugWarning:
         "This modification causes a change in the address of the debate. Please take this change into account in further communications. To insure the continuity of the service, the previous address will still be operating.",
       invalidSlug: "Special characters are not allowed",
+      theme: {
+        themeColorsTitle: "Theme colors",
+        firstColor: "Primary color",
```

Comment:
technically I would put a 'u' in colour but it depends if we're using american english or not

Reply:
I learnt english in the US @gary-bluenove and I am more of a burger guy than fish and chip :)

Reply:
![america-fuck-yeah-memes-com-13888225](https://user-images.githubusercontent.com/26386718/62277306-2a2fd900-b446-11e9-8665-8bb14f09b7ee.png)